flake.nix

```diff
@@ -51,6 +53,13 @@
           # Development shell configuration
           devShell = {
             hlsCheck.enable = false;
+            tools = _:
+              let devtools = inputs.horizon-devtools.legacyPackages.${system};
+              in {
+                inherit (devtools) ghcid haskell-language-server;
+                # FIXME: Make this available in horizon-devtools
+                hlint = null;
```

Comment:
@locallycompact Could you add `hlint` to horizon-devtools?

Reply:
It is there

```
nix build git+https://gitlab.horizon-haskell.net/package-sets/horizon-devtools#hlint
```

Reply:
Right, PEBKAC